modules/psoxy_connection/main.tf

```diff
@@ -0,0 +1,29 @@
+resource "local_file" "worklytics_tenant_api_script_file" {
```

Comment:
This is going to create the file to be executed, but then we need to run it ourselves right?

Something it happens when I was doing [workload identity in psoxy](https://github.com/Worklytics/psoxy/pull/242/commits/495cfa9b7bd1d0e5ce7b9cb9060ba18c99d6816b#r1088197220).

I had to put the file as external and invoke it through `data` . With that, the script is executed as part of apply -and users don't need to do more interactions with TF-